resources/templates/index.twig.html

```diff
@@ -18,14 +18,14 @@
     <body class="h-full flex flex-col font-sans {{ bodyBackground  ? '' : 'body-gradient-' ~ themeColour|lower }}" style="{{ bodyBackground  ? bodyBackground : '' }} {{ themeColour is empty ? 'background: radial-gradient(80% 1000px at 20% 500px, #ef99c7 0%, #794d95 100%) no-repeat fixed ;' : '' }}">
 
         {% set sidebarPos = isLoggedIn ? 'left' : 'right' %}
-        
+
         <div class="px-4 sm:px-6 lg:px-12 pb-24">
-        
+
             {% block header %}
                 <div id="header" class="relative flex justify-between items-center">
 
                     <a id="header-logo" class="block my-4 max-w-xs sm:max-w-full leading-none" href="{{ absoluteURL }}">
-                        <img class="block max-w-full {{ isLoggedIn ? 'h-12' : 'h-20 mt-4 mb-4' }}" alt="{{ organisationName }} Logo" src="{{ absoluteURL }}/{{ organisationLogo|default("/themes/Default/img/logo.png") }}" style="max-height:100px;" />
+                        <img class="block max-w-full {{ isLoggedIn ? 'h-12' : 'h-20 mt-4 mb-4' }}" alt="{{ organisationName }} Logo" src="{{ absoluteURL }}{{ organisationLogo|default("/themes/Default/img/logo.png") }}" style="max-height:100px;" />
```

Comment:
The absoluteURL and absolutePath are always saved without trailing slashes, so this would break existing installs. Perhaps double-check that the refactored installer is also `rtrim`ing trailing slashes, for consistency.

Reply:
Although the absoluteURL and absolutePath are stored without trailing slashes, the logo path here has a leading slash. So there is no need for an additional trailing slash to the absoluteURL.

Reply:
Hmm, shoot, that must be a bug then from when the field was changed from text to a file uploader 😬 Existing installs do not have a leading slash on this setting

<img width="495" alt="Screenshot 2021-09-30 at 2 50 49 PM" src="https://user-images.githubusercontent.com/897700/135401900-c98f99b9-5f0e-4a89-8b7b-e3992c8d41d7.png">

Reply:
Nevermind. Modified again to remove any leading slash from the logo path.